core/Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["network-programming", "asynchronous"]
 [dependencies]
 asn1_der = "0.6.1"
 bs58 = "0.3.0"
-ed25519-dalek = "1.0.1"
+ed25519-dalek = "1.0.0-pre.4"
```

Comment:
Why downgrade this dependency?

https://crates.io/crates/ed25519-dalek

Reply:
That's a rebase failure from my side, I'll fix it.